var/spack/repos/builtin/packages/bcl2fastq2/package.py

```diff
@@ -38,6 +38,7 @@ class Bcl2fastq2(Package):
 
     homepage = "https://support.illumina.com/downloads/bcl2fastq-conversion-software-v2-18.html"
```

Comment:
This homepage no longer exists. Can you update it to:
https://support.illumina.com/downloads/bcl2fastq-conversion-software-v2-20.html